webpack/scenes/Tasks/TaskActions.js

```diff
@@ -0,0 +1,75 @@
+import { foremanTasksApi as api } from '../../services/api';
+import { propsToSnakeCase } from '../../services/index';
+
+import {
+  GET_TASK_REQUEST,
+  GET_TASK_SUCCESS,
+  GET_TASK_FAILURE,
+  TASK_BULK_SEARCH_REQUEST,
+  TASK_BULK_SEARCH_SUCCESS,
+  TASK_BULK_SEARCH_FAILURE,
+} from './TaskConstants';
+
+export const bulkSearch = (extendedParams = {}) => (dispatch) => {
```

Comment:
These task related actions, reducers, and constants should probably be move to foreman (or at least the `move_to_foreman` folder 😄)